pip_review/__main__.py

```diff
@@ -244,55 +136,59 @@ def confirm(question):
     return answer == 'y'
 
 
+def get_outdated_packages(local=False, pre_release=False, user=False):
+    command = ['pip', 'list', '--outdated', '--disable-pip-version-check']
```

Comment:
For now, let's concentrate on getting this PR to work, and i will fix the travis build in an other PR ;)